databricks/koalas/tests/test_indexes.py

```diff
@@ -524,3 +524,145 @@ def test_argmax(self):
                 TypeError,
                 "reduction operation 'argmax' not allowed for this dtype"):
             kidx.argmax()
+
+    def test_monotonic(self):
+        # test monotonic_increasing & monotonic_decreasing for MultiIndex
+        kmidx = ks.MultiIndex.from_tuples(
+            [('w', 'a'), ('x', 'b'), ('y', 'c'), ('z', 'd')])
+        pmidx = kmidx.to_pandas()
```

Comment:
Could you add comments to describe each example is increasing or decreasing or none to be easy to read each case?

Reply:
Also could you add some cases including `None`?

Reply:
Btw, can we use just a loop? Seems like each test is the same shape.

Reply:
@ueshin ah, yes i'm working to make this shorten :)